docs/template-examples/helper-functions.md

```diff
@@ -72,8 +72,12 @@ requests:
         55: {{gzip_decode(hex_decode("1f8b08000000000000fff248cdc9c907040000ffff8289d1f705000000"))}}
         56: {{zlib("Hello")}}
         57: {{zlib_decode(hex_decode("789cf248cdc9c907040000ffff058c01f5"))}}
-        58: {{date("%Y-%M-%D")}}
-        59: {{time("%H-%M")}}
-        60: {{timetostring(unixtime())}}
-        61: {{compare_versions('v1.0.0', '>v0.0.1', '<v1.0.1')}}
+        58: {{compare_versions('v1.0.0', '>v0.0.1', '<v1.0.1')}}
```

Comment:
Just an FYI, we have this integration test, which was intended to also help with this documentation.

https://github.com/projectdiscovery/nuclei/blob/9fd9892f49fabb3fc836caad698d6ecdd8c7d551/integration_tests/http/dsl-functions.yaml#L9